Makefile

```diff
@@ -8,4 +8,4 @@ lint:
 
 lint-fix:
 	ruff check --fix
-	npx @biomejs/biome check --fix
+	npx @biomejs/biome check --fix
```

Comment:
Can you install Editorconfig and resave this file? Because now you removed the trailing newline that we want. https://editorconfig.org/